docs/ddp.md

```diff
@@ -0,0 +1,243 @@
+# How to do DistributedDataParallel
+
+This document shows how to use torch.nn.parallel.DistributedDataParallel in xla, and further describes its difference against the native xla data parallel approach.
+
+
+## Background / Motivation
+
+Customers have long requested the ability to use PyTorch’s DistributedDataParallel API with xla. And here we enable it as an experimental feature.
+
+
+## How to use DistributedDataParallel
+
+For those who switched from the PyTorch eager mode to XLA, here are all the changes you need to do to convert your eager DDP model into XLA model. We assume that you already know how to use XLA [on a single device](https://github.com/pytorch/xla/blob/master/API_GUIDE.md#running-on-a-single-xla-device).
```

Comment:
Should we use a relative link to the API_GUIDE here so it doesn't always go to master?